.github/workflows/dbld-images.yml

```diff
@@ -22,13 +22,18 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
-    env:
-      CONTAINER_REGISTRY: "ghcr.io/${{ github.repository_owner }}"
-  
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
 
+      - name: Set container registry
+        run: |
+          CONTAINER_REGISTRY="ghcr.io/${{ github.repository_owner }}"
+          CONTAINER_REGISTRY="$(echo "$CONTAINER_REGISTRY" | awk '{print tolower($0)}')"
```

Comment:
${CONTAINER_REGISTRY,,} is a bash expression that converts the value to lowercase. no need for external processes or awk.

```
bazsi@bzorp:~/src/syslog-ng/syslog-ng$ value=CamelCase
bazsi@bzorp:~/src/syslog-ng/syslog-ng$ echo ${value^^}
CAMELCASE
bazsi@bzorp:~/src/syslog-ng/syslog-ng$ echo ${value,,}
camelcase

```

Reply:
NOTE: this is bash related, so dash does not support it.

Reply:
I was planning to use the bash >=4.0 syntax you mentioned, but I thought POSIX compatibility would be better for us.

I can switch to this syntax, if it is preferred :)

Reply:
no need. I can see now that even old bash does not support it. I hate awk code within shell scripts, but this is still ok. I would probably have used tr '[A-Z]' '[a-z]' once I recognized the bash version constraints.

but like I said, this is fine.